<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGaugeWeather</class>
 <widget class="QWidget" name="frmGaugeWeather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="GaugeWeather" name="gaugeWeather" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="horizontalSlider1">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider2">
       <property name="minimum">
        <number>-50</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckAnimation">
       <property name="text">
        <string>动画</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>晴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>雨</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>雪</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>多云</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>雪雨</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>冰雹</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>闪电</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>雾</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>局部多云</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugeWeather</class>
   <extends>QWidget</extends>
   <header>gaugeweather.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
